lib/ttnt/test_selector.rb

```diff
@@ -29,6 +32,8 @@ def select_tests
       diff = @base_obj.diff(@target_obj)
       diff.each_patch do |patch|
         file = patch.delta.old_file[:path]
+        tests << file and next if test_file?(file)
```

Comment:
Seems nice, thank you! I avoided it since logic in else block seemed a little too long to me. So, I made if-else explicit and moved the logic in else block to another method to make it more readable: https://github.com/Genki-S/ttnt/commit/a5f1fb61d74a114692a7b05f50feeeda86222a88